db/migrations/create_database_and_tables.sql

```diff
@@ -0,0 +1,12 @@
+CREATE DATABASE chitter_manager;
+CREATE TABLE peeps(id SERIAL PRIMARY KEY, text VARCHAR(140), handle VARCHAR(50), user_name VARCHAR(60));
+ALTER TABLE peeps ADD time_posted TIMESTAMP;
```

Comment:
This those SQL statements are part of the same migration, they'll always be executed together — so it's fine to move the "ALTER TABLE" new column to be part of the initial "CREATE TABLE" so the table is created with the `time_posted` column initially too :)